src/state.js

```diff
@@ -1,3 +1,12 @@
-const INITIAL_STATE = {};
+const INITIAL_STATE = {
+	locations: [],
```

Comment:
Please update your text editor to do 2 spaces per tab, and retab this out so the spacing is 2 spaces.